frontend/src/pages/register/formValidation.tsx

```diff
@@ -113,7 +124,12 @@ const FormValidation = () => {
             {...register(fieldConfig.name, fieldConfig.validations)}
           />
         ))}
-        <CheckBox register={register} error={errors.areTermsRead?.message} />
+        <CheckBox
+          label={tos}
```

Comment:
Would it maybe be better to use "children" for label? That way you wouldn't have to store "tos" in a variable.

Reply:
Yes, now it's with children.